Dx diagram new api descriptions (#416)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/onCustomCommandExecuted.md

---
---
##### shortDescription

> A function that is executed after a custom command item has been clicked and allows you to perform custom actions.

Reviewer: ```suggestion
A function that is executed after a custom command item was clicked and allows you to perform custom actions.
```

Reviewer: See the comment below
